packages/react-native/Libraries/Text/Text.js

```diff
@@ -163,6 +163,17 @@ const Text: component(
         // $FlowFixMe[incompatible-type]
         _style = [_style, overrides];
       }
+
+      // [macOS
+      // For some reason, on macOS, Setting backgroundColor without borderRadius on Text will
+      // cause the color to spill pass the frame of the view. We can solve this by setting a dummy
+      // value for borderRadius.
+      if (Platform.OS === 'macos') {
+        if ('backgroundColor' in _style && !('borderRadius' in _style)) {
+          _style.borderRadius = 0.001;
```

Comment:
non-blocking: Does 0 work here, or does this have to be a nonzero value?

Reply:
If this is meant to be a symbolic "filler" value, perhaps you could try `Number.MIN_VALUE` so it's the smallest non-zero available?

Reply:
Oh, that's a good idea. I'll try!